ui-core/src/components/inputs/TimePicker.tsx

```diff
@@ -74,12 +124,16 @@ const TimePicker = ({ onTimeChange, hours = 0, minutes = 0, ...otherProps }: Tim
   return (
     <div className="time-picker">
       <Input
+        className={cx('input', 'time-input', {
+          'time-input-firefox': navigator.userAgent.includes('Firefox'),
```

Comment:
What do you think about applying the style of `time-input-firefox` class using the css rules `@-moz-document url-prefix()`? For example :
```css
@-moz-document url-prefix() {
  .time-picker .time-input {
    letter-spacing: -0.9px;
  }
}
```
This way, you wouldn’t need to add a specific class for Firefox, and the style would only apply to Firefox without changing the component.

Reply:
it’s much better ! done